lagunita/lms/templates/static_templates/contact.html

```diff
@@ -13,13 +13,9 @@
 <%block name="pagetitle">${_('Contact')}</%block>
 
 <main id="main" aria-label="Content" tabindex="-1">
-    <section class="container about contact">
+    <section class="container contact">
         <section class="vision">
             <section class="message left" id="contact">
-                <div class="photo">
```

Comment:
why are removing the dish photo?

Reply:
Eh, why not?
The about page looks nice spruced up w/ an image, but this is just contact info, so one less distraction?
I mean, we could include it inline, but doesn't feel like it adds much to the page. It's cute and all, the dish for contact ;)